### DISCUSSION TITLE
ddns添加NOIP访问

### DISCUSSION THREAD

**Discussion by furry20110108** [jeessy2/ddns-go#1293] [Ideas]
windows的选项加一个no-ip的吧，我的是no-ip的
密钥主机：all.ddnskey.com
DDNS服务器 ：www.no-ip.com
用户要填主机名，账户，密码
账户有时候不是邮件
![7824054e-288f-4084-8dbf-b016d785fc86](https://github.com/user-attachments/assets/d36af81c-db5d-4ef6-b7e1-2814abe069f4)

**Comment by wangchen82288** [comment]
加一个noip吧
